tests/e2e/utils/constants.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+const (
+	CLIBinary   = "./bin/avalanche"
```

Comment:
this seems duplicated with the other constants file. besides that, why not a unique constants for all e2e?

Reply:
I deleted the duplicate. I don't think there are a lot of wide spanning constants that are useful in multiple packages. Most of the use cases are contained, so I didn't break it out into its own package. We can add it later if we want.